### DISCUSSION TITLE
Rename app asset on macOS to "Thorium Reader.app"?

### DISCUSSION THREAD

**Discussion by ylluminate** [edrlab/thorium-reader#2682] [Developer corner]
I've noticed that when attempting to install Thorium Reader and the Thorium Browser there is a conflict in names. Any chance we can have the app renamed to "Thorium Reader.app" on macOS so that Homebrew can more readily manage it and we can reduce conflicts?

**Comment by danielweck** [comment]
Related: https://github.com/edrlab/thorium-reader/discussions/2638
